CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## master / unreleased
 
 * [CHANGE] Ingester: don't update internal "last updated" timestamp of TSDB if tenant only sends invalid samples. This affects how "idle" time is computed. #3727
+* [ENHANCEMENT] Prometheus upgraded. #3739
+  * `@` modifier in PromQL.
```

Comment:
Since this is disabled by default, I would not mention it and add that mention to the CHANGELOG once will be enabled. What do you think?

Reply:
Sounds good